cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/views/home/index.js

```diff
@@ -0,0 +1,51 @@
+import { useCallback } from "react";
+import { useHistory, Link } from "react-router-dom";
+import Hero from "../../components/hero";
+import Button from "../../components/button";
+import { useScrollToTop } from "../../components/scroll-to-top";
+
+import heroImg from "img/Hero_2.png";
+import arrowRight from "@cfpb/cfpb-icons/src/icons/arrow-right.svg";
+
+export default function Home() {
+  useScrollToTop();
+
+  const history = useHistory();
+
+  const nextPage = useCallback(
```

Comment:
You don't need this, or the `useHistory` hook if all you need is a `Link` component